libs/data-access/src/lib/api/concept/index.ts

```diff
@@ -9,7 +9,7 @@ export const searchConceptsForCatalog = async (
       Authorization: `Bearer ${accessToken}`,
       'Content-Type': 'application/json',
     },
-    body: JSON.stringify({query}),
+    body: query,
```

Comment:
feks noe i denne duren
```suggestion
    body: jsonSearchBody,
```